StreamDeckPlugin/Utils/TextUtils.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Text;
+
+namespace StreamDeckPlugin.Utils {
+    public static class TextUtils {
+        public static string WrapTitle(string title) {
+            string[] words = title.Split(' ');
+
+            var newSentence = new StringBuilder();
+            var line = "";
+            foreach (string word in words) {
+                if ((line + word).Length > 10) {
```

Comment:
minor: if the word is over 10 characters long, something weird will happen

Reply:
Are you sure? I think the larger than 10 character word would just get its own line, yes? It will get added at line 16 and then fail the check at 11 and get appened.

(note: this is code I totally just copied off the internet- lol)